langkit/gdb/tdh.py

```diff
@@ -65,7 +65,10 @@ def kind(self):
 
     @property
     def sloc_range(self):
-        return SlocRange(self.value['sloc_range'])
+        sloc_range_sym = "langkit_support__token_data_handlers__sloc_range"
+        return SlocRange(gdb.lookup_global_symbol(sloc_range_sym).value()(
+            self.tdh.value, self.value
+        ))
```

Comment:
Pretty-printers should not try to execute inferior code, both for performance and to avoid semi-automatic process corruption when data structures are inconsistent. I’m afraid we need to re-implement the Sloc_Range method here.